src/blockchain/block.js

```diff
@@ -103,38 +102,33 @@ export function verifyBlock (data, validators, networkId) {
     }
 
     if (precommit.network_id !== networkId ||
-            precommit.protocol_version !== PROTOCOL_VERSION ||
-            precommit.service_id !== CORE_SERVICE_ID ||
-            precommit.message_id !== PRECOMMIT_MESSAGE_ID) {
+      precommit.protocol_version !== PROTOCOL_VERSION ||
+      precommit.service_id !== CORE_SERVICE_ID ||
+      precommit.message_id !== PRECOMMIT_MESSAGE_ID) {
       return false
     }
 
     if (precommit.body.height !== data.block.height) {
-            // wrong height of block in precommit
+      // wrong height of block in precommit
       return false
     } else if (precommit.body.block_hash !== blockHash) {
-            // wrong hash of block in precommit
+      // wrong hash of block in precommit
       return false
     }
 
     if (round === undefined) {
       round = precommit.body.round
     } else if (precommit.body.round !== round) {
-            // wrong round in precommit
+      // wrong round in precommit
       return false
     }
 
-    var publicKey = validators[precommit.body.validator]
+    const publicKey = validators[precommit.body.validator]
 
     if (!verifySignature(precommit.signature, publicKey, precommit.body, Precommit)) {
       return false
     }
   }
 
-  if (uniqueValidators.length <= validatorsTotalNumber * 2 / 3) {
-        // not enough precommits from unique validators
-    return false
-  }
-
-  return true
+  return uniqueValidators.length > validatorsTotalNumber * 2 / 3
```

Comment:
this option is certainly shorter but previous one was a bit more understandable...